### PR TITLE
fix(tests): Fix `<Item>` `textValue` warnings

### PR DESCRIPTION
**Pull request by vuluongj20**
Fix all those `textValue` warnings we've been seeing a lot of lately:
Screenshot 2023-02-14 at 12 19 10 PM

They all come from just two components: `Tabs` and charts' `OptionSelector`.

### CODE REVIEW

File: static/app/components/charts/optionSelector.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ function OptionSelector({
   const mappedOptions = useMemo(() => {
     return options.map(opt => ({
       ...opt,
+      textValue: opt.label,
```
Comment (malwilley) [inline static/app/components/charts/optionSelector.tsx:50]: Looks like you have to cast to string here so TS will be happy?

Comment (vuluongj20) [reply static/app/components/charts/optionSelector.tsx]: Oh true somehow I missed that! Just fixed it

File: static/app/components/charts/optionSelector.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ function OptionSelector({
   const mappedOptions = useMemo(() => {
     return options.map(opt => ({
       ...opt,
+      textValue: String(opt.label),
```
Comment (evanpurkhiser) [inline static/app/components/charts/optionSelector.tsx:50]: Would `toString` work?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/charts/optionSelector.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/components/charts/optionSelector.tsx:50]: We already use `String(opt.label)` beneath as the text value for `<Truncate />`, so I'm doing the same here.

**Pull request merged**